test/e2e/angular-app/package.json

```diff
@@ -20,7 +20,7 @@
     "@angular/http": "^5.0.0",
     "@angular/platform-browser": "^5.0.0",
     "@angular/platform-browser-dynamic": "^5.0.0",
-    "@angular/router": "^5.0.0",
+    "@angular/router": "5.2.7",
```

Comment:
For more context - this issue was discovered in [okta-oidc-js](https://github.com/okta/okta-oidc-js/issues/140) and extends down to AuthJS.